src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -30,6 +31,7 @@ export const TriggerViewContext = createContext({
     selectMaterial: (materialId) => { },
     toggleChanges: (materialId: string, hash: string) => { },
     toggleInvalidateCache: () => { },
+    fetchMaterialByCommit: (ciNodeId: number, pipelineName: string, materialId: number, commitHash: string) => { },
```

Comment:
We usually use the prefix 'get' while fetching data instead of 'fetch'. what you say

Reply:
Done